_emails/us/oregon/washington_county.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Washington County, OR"
+permalink: "/washington_county"
```

Comment:
If you merge in the latest master, you'll see that tests will fail for this permalink

Reply:
ahh ok, got it - messed up the merge, so opened up a new pr at #1427